setup.py

```diff
@@ -1,32 +1,30 @@
-from setuptools import setup, Extension
-import platform
+import setuptools
 
 version = '2.9.1'
```

Comment:
Should version be updated?

Reply:
Probably yes, when a new release is made. But that's your call :-)

Reply:
:-) @peterbarker 's